test/unit/component/registry/bpmn-model-registry.test.ts

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 import type { EdgeBpmnSemantic, ShapeBpmnSemantic } from '@lib/component/registry';
 import { BpmnModelRegistry } from '@lib/component/registry/bpmn-model-registry';
-import { expectAssociationFlow, expectLane, expectMessageFlow, expectPool, expectSequenceFlow, expectStartEvent } from '../../helpers/bpmn-semantic-utils';
+import { expectAssociationFlow, expectLane, expectMessageFlow, expectPool, expectSequenceFlow, expectStartEvent } from '@test/shared/model/bpmn-semantic-utils';
 import { associationFlowInModel, laneInModel, messageFlowInModel, poolInModel, sequenceFlowInModel, startEventInModel } from '../../helpers/bpmn-model-utils';
```

Comment:
**suggestion:** 
```suggestion
import { associationFlowInModel, laneInModel, messageFlowInModel, poolInModel, sequenceFlowInModel, startEventInModel } from '@test/shared/model/bpmn-model-utils';
```

Reply:
❌ Not possible: the file is in `test/unit/helpers/bpmn-model-utils` not in `test/shared`.
I only moved the code that is currently shared between different kind of tests, not all the utils/helpers code.